ROADMAP.md

```diff
@@ -0,0 +1,24 @@
+# Roadmap of MPI Operator
+
+This document provides a high-level overview of where MPI Operator will grow in future releases. See discussions in the original RFC [here](https://github.com/kubeflow/mpi-operator/pull/159).
+
+## New Features / Enhancements
+
+* Decouple the tight dependency on Open MPI and support other collective communication frameworks.
+Related issue: [#12](https://github.com/kubeflow/mpi-operator/issues/12).
+* Support new versions of MPI Operator in [kubeflow/manifests](https://github.com/kubeflow/manifests).
+* Resign different components of MPI Operator to support fault tolerant collective communication frameworks such as [caicloud/ftlib](https://github.com/caicloud/ftlib).
```

Comment:
/cc @carmark 

If you are interested in it.

Reply:
@gaocegege Thanks, very interested in it.

Reply:
I think we already implement both features in ftlib

/cc @zw0610

Reply:
Our fault-tolerant design can handle both situations. However, so far, we do not consider any MPI implementations. The collective communication is handled by other libraries such like NCCL and Gloo.